src/objects/Course_Offering_Schedule__c.object

```diff
@@ -55,7 +55,7 @@
     <enableStreamingApi>true</enableStreamingApi>
     <fields>
         <fullName>Course_Offering__c</fullName>
-        <deleteConstraint>Restrict</deleteConstraint>
+        <deleteConstraint>SetNull</deleteConstraint>
```

Comment:
Nice work calling this out in release notes, do we want to explicitly mention that with this change the parent record is deleted and the lookup field is cleared coz we are changing the existing functionality?

Reply:
@skosgi1 , sorry I might be confusing or missing something, are you referring to the changes section? To include a more explicit explanation of why? I have added "We have removed the platform's standard deletion so that customers can have the control of preventing deletion via Hierarchy Settings." as part of the explanation but let me know if I should provide more details?

Reply:
Sorry I wasn't clear, I thought we could add something like this to what you already have.. 'with this change, the lookup field on the child record is cleared if the parent is deleted'. Let me know if this makes sense @kdy618.

Reply:
@skosgi1 , thank you :) I've updated it above. Please let me know if I need to edit it more.

Reply:
I think it is course offering right? With this change, the Course Offering lookup field on the Course Offering schedule record is cleared if the Course Offering is deleted. @kdy618

Reply:
@skosgi1 , ahhh you're right! I'm sorry. I was also editing the Contact prevent deletion one.

Reply:
haha no worries. Looks great! Moving the WI along.